src/App.tsx

```diff
@@ -107,7 +109,8 @@ export const App: React.FC = () => {
       = comments.filter(comment => comment.id !== commentId);
 
     setComments(filteredComments);
-    deleteComment(commentId);
+    deleteComment(commentId)
+      .catch(() => setIsCommentError(true))
```

Comment:
here you need to make sure that your state is filtered only with a successful deletion request, otherwise now you first filter the state and only then ask the server if it can do this